protostar/cheatable_starknet/controllers/transaction_info.py

```diff
@@ -39,9 +39,8 @@ def get_for_contract(self, contract_address: Address) -> TxInfo:
             transaction_hash=self.state.contract_address_to_transaction_hash.get(
                 contract_address, 0
             ),
-            # TODO verify if this is a good default value
             chain_id=self.state.contract_address_to_chain_id.get(
-                contract_address, StarknetChainId.TESTNET.value
+                contract_address, DEFAULT_CHAIN_ID
```

Comment:
Changes in this file are confusing, could you please explain what is happening here? This PR explicitly says "docs" and these do not look like docs to me. Are you solving some kind of separate issue here?

Reply:
This a small readability improvement, nothing changed because `DEFAULT_CHAIN_ID = StarknetChainId.TESTNET.value` in `starkware.starknet.definitions.general_config`

Reply:
and the comment two lines up is removed because this is a good default value